main_test.go

```diff
@@ -22,6 +23,10 @@ import (
 var (
 	nopGoogleClient = &google.Client{}
 	nopZoomClient   = &zoom.Client{}
+	rp              = runParams{
```

Comment:
I wonder what it used before as deafults?

Reply:
Looks like prior to https://github.com/graphaelli/zat/pull/2 the http server didn't take runParams since it couldn't trigger an archive action.

It gets created at https://github.com/graphaelli/zat/pull/2/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R492-R495 from cli flags, so I took the test defaults form there (https://github.com/matschaffer/zat/blob/4286d29878c568eced9f0f4b8d66abca84e1adaf/main.go#L503-L503)